scripts/utils.ts

```diff
@@ -24,6 +24,13 @@ export function shI(
   return spawnSync(command, args, { stdio: 'inherit', ...config });
 }
 
+/**
+ * Just like the `sh` function but prints all stdio output (verbose)
+ */
+export function shV(command: string) {
```

Comment:
Is this change going to break other scripts that are calling shVerbose?

Reply:
There was only one script that defined and used `shVerbose`, so I moved it to here and renamed it `shV` so it could be shared